### ISSUE TITLE
Old thread stays bumped after post was deleted.

### ISSUE THREAD

**Issue by exelotl** [nim-lang/nimforum#319]
Sometimes on the Nim forums I notice that an old thread has risen to the top again. I check it but there are no new posts.

I assume some spam comment brought it to the top, and the spam was deleted, but since the modification date of the thread has changed, it remains at the top.

I guess when the last message in a thread is deleted, the modification time should revert back to the timestamp of the new last message?

**Comment by dom96** [comment]
yep, this is a long-standing bug. Indeed, we need to update the thread's modification time to the last post in that thread.
